src/types/wallet.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Configuration interface for blockchain networks.
+ * Contains all necessary information to connect to and interact with a blockchain.
+ */
+export interface ChainConfig {
```

Comment:
This is chain configuration, but it's specifically for usage with wallet connections